install_files/securedrop-app-code/DEBIAN/postrm

```diff
@@ -19,14 +19,20 @@ set -e
 # the debian-policy package
 . /usr/share/debconf/confmodule
 
+clean_pyc() {
+    find /var/www/securedrop/alembic -name '*.pyc' -delete
```

Comment:
`lintian` disagrees with you. [It says](https://lintian.debian.org/tags/command-with-path-in-maintainer-script.html) we *shouldn't* prepend the full path because `$PATH` is "set to a reasonable value."